Cognigy/WebchatWidget
# feature/20924 upgrade react and emotion

How to test:

Run all tests and check if all plugins still work.

@emotion/cache is still v10 because v11 has breaking changes to the plugin system
